### ISSUE TITLE
Input type file uses white outline on focus in general settings page

### ISSUE THREAD

**Issue by Ifycode** [ushahidi/platform#4149]
Based on the fix I made here in [PR 259](https://github.com/ushahidi/platform-pattern-library/pull/259) that should affect all buttons on the site, button elements that should have been skipped now receive focus - this is the reason why the buttons are not skipped in the screenshot below.

**Describe the bug:**

Input type file has a white outline when in focus instead of the one applied to other input types.

**Screenshot:**

![Site Settings - Obiagba Mary](https://user-images.githubusercontent.com/45185388/97819689-17b2d980-1c5f-11eb-8fad-8ac5fa48dc2a.gif)

**Where was the bug observed:**

http://localhost:3000/settings/general

**Steps to reproduce the behavior:**
1. Login to your account.
2. Go to setting > general.
3. Navigate to the input type file with the tab key.

**Suggestions:**
- I suggest the same styles applied to the other inputs/textarea should be applied to input type file too (that is if there's no other plan for input type file).
- Also, what if the generated API key is also made to receive focus?

Aha! Link: https://ushahiditeam.aha.io/features/PROD-893
